cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -160,6 +160,20 @@ def __init__(self, app, instrument_name=None, par_dic=None, config=None, data_se
             else:
                 logger.debug("NOT get_meta_data request: yes scratch_dir")
 
+                self.set_sentry_client()
+                if data_server_call_back is False:
```

Comment:
at least pep8 does not encourage this bool comparison https://www.python.org/dev/peps/pep-0008/